Fails to build for arm-unknown-linux-musl

```
error[E0308]: mismatched types                                                                                                                        
  --> /home/vi/.cargo/registry/src/github.com-1ecc6299db9ec823/input-linux-0.0.1/src/time.rs:26:21                                                    
   |                                                                                                                                                  
26 |             tv_sec: secs,                                                                                                                        
   |                     ^^^^ expected i32, found i64                                                                                                 
                                                                                                                                                      
error[E0308]: mismatched types                                                                                                                        
  --> /home/vi/.cargo/registry/src/github.com-1ecc6299db9ec823/input-linux-0.0.1/src/time.rs:27:22                                                    
   |                                                                                                                                                  
27 |             tv_usec: usecs,                                                                                                                      
   |                      ^^^^^ expected i32, found i64                                                                                               
                                                                                                                                                      
error[E0308]: mismatched types                                                                                                                        
  --> /home/vi/.cargo/registry/src/github.com-1ecc6299db9ec823/input-linux-0.0.1/src/time.rs:36:9                                                     
   |                                                                                                                                                  
35 |     pub fn seconds(&self) -> i64 {                                                                                                               
   |                              --- expected `i64` because of return type                                                                           
36 |         (self.0).tv_sec                                                                                                                          
   |         ^^^^^^^^^^^^^^^ expected i64, found i32                                                                                                  
help: you can cast an `i32` to `i64`, which will sign-extend the source value                                                                         
   |                                                                                                                                                  
36 |         (self.0).tv_sec.into()                                                                                                                   
   |                                                                                                                                                  
                                                                                                                                                      
error[E0308]: mismatched types                                                                                                                        
  --> /home/vi/.cargo/registry/src/github.com-1ecc6299db9ec823/input-linux-0.0.1/src/time.rs:40:27                                                    
   |                                                                                                                                                  
40 |         (self.0).tv_sec = value                                                                                                                  
   |                           ^^^^^ expected i32, found i64                                                                                          
                                                                                                                                                      
error[E0308]: mismatched types                                                                                                                        
  --> /home/vi/.cargo/registry/src/github.com-1ecc6299db9ec823/input-linux-0.0.1/src/time.rs:44:9                                                     
   |                                                                                                                                                  
43 |     pub fn microseconds(&self) -> i64 {                                                                                                          
   |                                   --- expected `i64` because of return type                                                                      
44 |         (self.0).tv_usec                                                                                                                         
   |         ^^^^^^^^^^^^^^^^ expected i64, found i32                                                                                                 
help: you can cast an `i32` to `i64`, which will sign-extend the source value                                                                         
   |                                                                                                                                                  
44 |         (self.0).tv_usec.into()                                                                                                                  
   |                                                                                                                                                  
                                                                                                                                                      
error[E0308]: mismatched types                                                                                                                        
  --> /home/vi/.cargo/registry/src/github.com-1ecc6299db9ec823/input-linux-0.0.1/src/time.rs:48:28                                                    
   |                                                                                                                                                  
48 |         (self.0).tv_usec = value                                                                                                                 
   |                            ^^^^^ expected i32, found i64                                                                                         
                                                                                                                                                      
error: aborting due to 6 previous errors                                                                                                              
                                                                                                                                                      
For more information about this error, try `rustc --explain E0308`.                                                                                   
error: Could not compile `input-linux`.                                                                                                               

To learn more, run the command again with --verbose.
```

```
error[E0277]: the trait bound `i32: std::convert::From<i64>` is not satisfied                                                                         
  --> /mnt/src/git/input-linux-rs/src/time.rs:26:26                                                                                                   
   |                                                                                                                                                  
26 |             tv_sec: secs.into(),                                                                                                                 
   |                          ^^^^ the trait `std::convert::From<i64>` is not implemented for `i32`                                                   
   |                                                                                                                                                  
   = help: the following implementations were found:                                                                                                  
             <i32 as std::convert::From<i16>>                                                                                                         
             <i32 as std::convert::From<i8>>                                                                                                          
             <i32 as std::convert::From<u16>>                                                                                                         
             <i32 as std::convert::From<u8>>                                                                                                          
           and 3 others                                                                                                                               
   = note: required because of the requirements on the impl of `std::convert::Into<i32>` for `i64`                                                    
                                                                                                                                                      
error[E0277]: the trait bound `i32: std::convert::From<i64>` is not satisfied                                                                         
  --> /mnt/src/git/input-linux-rs/src/time.rs:27:28                                                                                                   
   |                                                                                                                                                  
27 |             tv_usec: usecs.into(),                                                                                                               
   |                            ^^^^ the trait `std::convert::From<i64>` is not implemented for `i32`                                                 
   |                                                                                                                                                  
   = help: the following implementations were found:                                                                                                  
             <i32 as std::convert::From<i16>>                                                                                                         
             <i32 as std::convert::From<i8>>                                                                                                          
             <i32 as std::convert::From<u16>>                                                                                                         
             <i32 as std::convert::From<u8>>                                                                                                          
           and 3 others                                                                                                                               
   = note: required because of the requirements on the impl of `std::convert::Into<i32>` for `i64`                                                    
                                                                                                                                                      
error: aborting due to 2 previous errors                                                                                                              
                                                                                                                                                      
For more information about this error, try `rustc --explain E0277`.                                                                                   
error: Could not compile `input-linux`.                                                                                                               

To learn more, run the command again with --verbose.
```

--- a/src/time.rs
+++ b/src/time.rs
@@ -2,6 +2,7 @@ use std::{fmt, cmp};
 use std::hash::{Hash, Hasher};
 use std::ops::{Deref, DerefMut};
 use sys::timeval;
+use nix::libc::{time_t,suseconds_t};
 
 #[repr(C)]
 #[derive(Copy, Clone)]
@@ -23,8 +24,8 @@ struct TimevalDef {
 impl EventTime {
     pub fn new(secs: i64, usecs: i64) -> Self {
         EventTime(timeval {
-            tv_sec: secs,
-            tv_usec: usecs,
+            tv_sec: secs as time_t,
+            tv_usec: usecs as suseconds_t,
         })
     }
 
@@ -33,19 +34,19 @@ impl EventTime {
     }
 
     pub fn seconds(&self) -> i64 {
-        (self.0).tv_sec
+        (self.0).tv_sec as i64
     }
 
     pub fn set_seconds(&mut self, value: i64) {
-        (self.0).tv_sec = value
+        (self.0).tv_sec = value as time_t
     }
 
     pub fn microseconds(&self) -> i64 {
-        (self.0).tv_usec
+        (self.0).tv_usec as i64
     }
 
     pub fn set_microseconds(&mut self, value: i64) {
-        (self.0).tv_usec = value
+        (self.0).tv_usec = value as suseconds_t
     }
 
     pub fn into_inner(self) -> timeval {

